pytorch_lightning/trainer/data_loading.py

```diff
@@ -51,6 +51,7 @@ class TrainerDataLoadingMixin(ABC):
     #  the proper values/initialisation should be done in child class
     val_check_interval: float
     tpu_local_core_rank: int
+    hpu_local_core_rank: int
```

Comment:
can we just combine those to local core rank? I don't think there is a need to have two arguments here that basically do the same but likely are only used mutually exclusively